RecordCollection/Views/Albums/Show.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model AlbumViewModel
+
+<h1>Album Details</h1>
+
+<div>
+<h2>@Model.Album.Title</h2>
+<p>Artist: @Model.Album.Artist</p>
+<p>Release Year: @Model.Album.ReleaseDate</p>
+
+@if (Model.IsModernAlbum())
+{
+<p>This album is recent.</p>
+}
+else
+{
+<p>This album is not recent.</p>
+}
+
+<h3>@Model.RelatedAlbumsCount Related Albums</h3>
```

Comment:
We could add an if statement here that checks if there are no related albums and displays "No related albums found" so the user doesn't wonder if it worked.